ui/vty-lib/src/Ariadne/UI/Vty/Widget/Wallet.hs

```diff
@@ -346,17 +346,7 @@ handleWalletWidgetEvent = \case
                     WalletAccount idx (fromMaybe "" uaciLabel) uaciBalance False)
                 (zip [0..] uwiAccounts)
           updateEditable walletAccountsL walletAccountsEditL converted
-          case uwiBalance of
-            Just balance ->
-              walletBalanceL .= BalanceResultSuccess balance
-            Nothing -> do
```

Comment:
AFAIU, this logic is still used in DSCP, will it be possible to migrate DSCP to this version without touching this code?